app/templates/new-signup.hbs

```diff
@@ -0,0 +1,28 @@
+{{page-title "New Sign Up"}}
+
+{{#if this.state}}
```

Comment:
Alternatively, what are your thoughts about this using "Contextual Components"? - [Reference](https://guides.emberjs.com/v3.1.0/components/wrapping-content-in-a-component/#toc_sharing-component-data-with-its-wrapped-content)

Reply:
Will track this in a different issue - https://github.com/Real-Dev-Squad/website-my/issues/248